tasks/security_agent.py

```diff
@@ -57,13 +58,21 @@ def build(
     go_mod="mod",
     skip_assets=False,
     static=False,
+    bundle=True,
 ):
     """
     Build the security agent
     """
+    if bundle and sys.platform != "win32":
+        return agent_build(
```

Comment:
I don't fully understand all the code in the tasks folder, so my question comes from a totally nobby perspective.

Should we be passing the flavor into `agent_build`?

Reply:
I'm not super familiar with the `ŧasks` forlder neither :-)
The flavor is indeed passed to `agent_build` when it's available (in trace_agent.py for instance, there is a `flavor` argument that is passed to `agent_build`, but for `security_agent.py` there is no such argument so it uses the default value for `flavor`. Does it make sense ?

Reply:
It does make sense. I left another comment regarding the default value https://github.com/DataDog/datadog-agent/pull/19280/files#r1442645399. Let me know what you think 😄